lib/uppy/job/loader.ex

```diff
@@ -1,2 +1,7 @@
 defmodule Uppy.Job.Loader do
+  alias Uppy.{Repo, Job}
+
+  def load_all do
+    Repo.all(Job)
```

Comment:
I was thinking about loading the jobs from a config file still, so you don't need an admin interface at all. WDYT?

So I guess on startup we should update the jobs table to reflect what's in the config table.